src/CMakeLists.txt

```diff
@@ -18,6 +18,7 @@ find_package(Boost ${requiredBoostVersion} REQUIRED)
 find_package(LibtorrentRasterbar ${requiredLibtorrentVersion} REQUIRED)
 find_package(OpenSSL ${requiredOpensslVersion} REQUIRED)
 
+add_definitions(-DBOOST_ALL_NO_LIB)
```

Comment:
maybe move this to cmake\Modules\winconf.cmake?

Reply:
Ok, done.